lib/ansible_tower_client/inventory_source.rb

```diff
@@ -0,0 +1,21 @@
+module AnsibleTowerClient
+  class InventorySource < BaseModel
+    def self.endpoint
+      "inventory_sources".freeze
+    end
+
+    def can_update?
```

Comment:
Would `updatable?` be a better name?  <-- Just a thought, not saying you have to change it if you don't like it